docs/fides/docs/quickstart/docker.md

```diff
@@ -97,22 +97,14 @@ volumes:
 
 Now we can start interacting with our installation. Let's run the following commands to get going:
 
-1. `docker-compose up -d` -> This will spin up the docker-compose file in the background
-1. `docker-compose run --rm fidesctl /bin/bash` -> Opens a shell within the fidesctl container
-1. `fidesctl ping` -> This confirms that your `fidesctl` CLI can reach the server and everything is ready to go!
-
-    ```bash
-    root@796cfde906f1:/fides/fidesctl# fidesctl ping
-    Pinging http://fidesctl:8080/health...
-    {
-        "data": {
-            "message": "Fidesctl API service is healthy!"
-        }
-    }
-    ```
```

Comment:
Should we restore this using `fidesctl status` and the updated response?

Reply:
eh...im indifferent. @conceptualshark do you think having the `ping` (now `status`) commands in there were meaningful/helpful? I can see it teaching users how to debug early on, but since it runs automatically anyway now...

Reply:
If `fidesctl status` can help with seeing any immediate issues if the installation does have problems, I like adding it in - even if it's the same as `ping` was. If you're following along step by step it can be nice to have that immediate verification that everything worked (even if it's a little extraneous).

Reply:
makes sense, I'll add it back in!